src/kv/key.rs

```diff
@@ -72,6 +72,12 @@ impl AsRef<Key> for kvrpcpb::Mutation {
     }
 }
 
+impl AsRef<Key> for kvrpcpb::LockInfo {
```

Comment:
Can we avoid this impl? It makes me feel a bit uncomfortable because the corresponding key is not the primary key.

Reply:
I'm sorry that I forgot to remove this block.
They are useless and just for debug.